### DISCUSSION TITLE
postman - only domain being scanned

### DISCUSSION THREAD

**Discussion by random-robbie** [blacklanternsecurity/bbot#1008] [General]
anyway to make it so postman stuff has to contain the domain that is being scanned? always find it's picking up stuff not related to the domain being scanned.

**Comment by TheTechromancer** [comment]
Can you give a specific example? Postman should only be returning results that contain the target domain in some form.

**Reply by random-robbie** [reply]
Try watsons.com.cn and lalamove.com 

Kept getting some app from Brasil jwts

**Reply by random-robbie** [reply]
I'll try find scan results and upload them tomorrow

**Reply by TheTechromancer** [reply]
Hmm, it appears postman's search is "smart". 

![Screenshot_20240120-175355.png](https://github.com/blacklanternsecurity/bbot/assets/20261699/7e8ddb2a-28af-430a-b231-3b979e69b630)

That's annoying.

@domwhewell-sage what do you think?

**Reply by TheTechromancer** [reply]
It would be nice if we could send a parameter that made the search literal, something like quotes for a Google search. If there isn't one we'll have to manually discard results that don't contain the query. Which actually wouldn't be too hard.

**Reply by domwhewell-sage** [reply]
I'm not sure there's a way to make the postman search literal.

@TheTechromancer I think discarding results is the way to go, at the moment all workspace are searched that are returned from the query. What if we only append the workspace ID if it contains the ORG_STUB?

**Reply by TheTechromancer** [reply]
Yeah I think that's the way to go. This will require a small rework, since it'll require us to make the requests inside the module instead of emitting the `UNVERIFIED_URL`s to httpx. (I suppose it wouldn't be the end of the world if we requested them twice, but I'd rather not)

**Comment by TheTechromancer** [comment]
Addressed in https://github.com/blacklanternsecurity/bbot/pull/1022.
